packages/contracts-bedrock/contracts/deployment/SystemDictator.sol

```diff
@@ -254,6 +254,22 @@ contract SystemDictator is OwnableUpgradeable {
                 )
             )
         );
+
+        // Dynamic config must be set before we can initialize the L2OutputOracle.
+        require(dynamicConfigSet, "SystemDictator: dynamic oracle config is not yet initialized");
+
+        // Upgrade and initialize the L2OutputOracle.
+        config.globalConfig.proxyAdmin.upgradeAndCall(
+            payable(config.proxyAddressConfig.l2OutputOracleProxy),
+            address(config.implementationAddressConfig.l2OutputOracleImpl),
+            abi.encodeCall(
+                L2OutputOracle.initialize,
+                (
+                    l2OutputOracleDynamicConfig.l2OutputOracleStartingBlockNumber,
+                    l2OutputOracleDynamicConfig.l2OutputOracleStartingTimestamp
+                )
+            )
+        );
```

Comment:
Actually, I don't think we can do this until after step2() when we set the DTL shut off block.

Perhaps we could make it something we do before `step3()`.